CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+# Contributing to DBOS-Docs
```

Comment:
Should we include text like "By submitting a pull request, you represent that you have the right to license your contribution to DBOS and the community, and agree by submitting the patch that your contributions are licensed under the MIT license."?